runtime/runtime/src/lib.rs

```diff
@@ -281,25 +281,32 @@ impl Runtime {
         debug!(target: "runtime", "{}", log_str);
     }
 
-    /// Validates all transactions in parallel and returns an iterator of
-    /// transactions paired with their validation results.
-    ///
-    /// Returns an `Iterator` of `(&SignedTransaction, Result<TransactionCost, InvalidTxError>)`
-    fn parallel_validate_transactions<'a>(
-        config: &'a RuntimeConfig,
+    fn parallel_validate_transactions(
+        config: &RuntimeConfig,
         gas_price: Balance,
-        transactions: &'a [SignedTransaction],
+        transactions: &[SignedTransaction],
```

Comment:
Nit: rather than cloning internally, have the caller do so. While it does mean that the callers who can't provide ownership of their list of signed transactions might have to allocate a new vector, the callers who are able to provide the ownership can avoid copies altogether.

(I would approach this from the angle of taking `impl IntoParallelIterator<Item=SignedTransaction>` as an argument.)